pkg/sentry/platform/kvm/bluepill_unsafe.go

```diff
@@ -210,8 +211,36 @@ func bluepillHandler(context unsafe.Pointer) {
 				return
 			}
 
-			c.die(bluepillArchContext(context), "exit_mmio")
-			return
+			// Increment the fault count.
+			atomic.AddUint32(&c.faults, 1)
```

Comment:
0bdd79ccd4695ba4539f2a288f88c60c8dd2ce1d has been reverted. They question is why it is needed.

Reply:
We added back as a work-around to make plugin stack workable since we met an issue caused by _cgo_sys_thread_start after including CGO. There is a more detailed explanation in https://github.com/google/gvisor/issues/9266#issuecomment-1855737336 and wonder whether there is any suggestion on how to solve this issue. Thanks a lot.

Reply:
Andrei's draft patch: https://github.com/google/gvisor/commit/c2ab4cb4d9daf501c09ae3ac3a624a78825d8c8d does solve this issue. Thanks a lot.